WordPress/src/jetpack/AndroidManifest.xml

```diff
@@ -3,7 +3,8 @@
     xmlns:tools="http://schemas.android.com/tools">
     <application>
         <!-- Deep Linking Activity -->
-        <activity android:name="org.wordpress.android.ui.deeplinks.DeepLinkingIntentReceiverActivity">
+        <activity android:name="org.wordpress.android.ui.deeplinks.DeepLinkingIntentReceiverActivity"
```

Comment:
> Updates Deeplink intent receiver activity exported attribute to true

Question (❓): I am guessing this is the last one of those, we just forgot to add the `android:exported` when this deep linking activity declaration got added to the Jetpack related `AndroidManifest.xml` back then, right?

PS: I then did a quick search based on `intent-filter` and didn't found any other activity, service or broadcast receiver to be missing this attribute, thus this seems 👍 , good job!

Reply:
>Question (❓): I am guessing this is the last one of those, we just forgot to add the android:exported when this deep linking activity declaration got added to the Jetpack related AndroidManifest.xml back then, right?

yup right. I missed it because it was in [jetpack/AndroidManifest.xml](https://github.com/wordpress-mobile/WordPress-Android/pull/17153/files/e62f406e38b6156f8ec964da1083577cc4e72802#diff-4fe6a8c900da11e5a97e3802bd447466245ba6ec8e51fe5a6572a063786a210f) and I only checked the main/AndroidManifest. 😅 

>PS: I then did a quick search based on intent-filter and didn't found any other activity, service or broadcast receiver to be missing this attribute, thus this seems 👍 , good job!

Thank you for doing the quick search. 👍🏼 . I had done the same but thanks for making sure. 👍🏼

Reply:
Awesome, thank you @AjeshRPai ! 🥇